### PR TITLE
fix: scripts for android and related doc

### LINKED ISSUE #707: Can't build on linux
```
yarn build  
yarn run v1.22.10
$ yarn install && npx pod-install
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
npx: installed 1 in 1.099s
Scanning for pods...
CocoaPods is only supported on darwin machines
```

### PR DESCRIPTION
**Pull request by Tbaut**
closes #707

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,13 +10,15 @@
   "scripts": {
     "android": "npx react-native run-android",
     "build-prod:ios": "NODE_OPTIONS=--max_old_space_size=8192 npx react-native bundle --dev false --entry-file index.js --bundle-output ios/main.jsbundle --platform ios --assets-dest='./ios'",
-    "build": "yarn install && npx pod-install",
+    "build:ios": "yarn install && npx pod-install",
```
Comment (kirushik) [inline package.json:13]: Maybe let's call it `install:ios` while we're at it?
It is really confusing to have a `build` target which doesn't, well, _build_ anything.

File: docs/wiki/Building-And-Publishing.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ In this page we will list the steps to publish the app on iOS App Store / Androi
 
 ##### Build production version
 
-1. Switch to the `master` branch and run `yarn build`
+1. Switch to the `master` branch and run `yarn install:ios`
```
Comment (TriplEight) [inline docs/wiki/Building-And-Publishing.md:9]:
```suggestion
1. Switch to the `master` branch and build the dependencies with `yarn install:ios`
```

**Review by TriplEight — CHANGES_REQUESTED** [review]
Looks good, except it needs an explanation what `yarn install:ios` does.

File: docs/wiki/Building-And-Publishing.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ In this page we will list the steps to publish the app on iOS App Store / Androi
 
 ##### Build production version
 
-1. Switch to the `master` branch and run `yarn build`
+1. Switch to the `master` branch and build the dependencies with `yarn install:ios`
```
Comment (Tbaut) [inline docs/wiki/Building-And-Publishing.md:9]:
```suggestion
1. Switch to the `master` branch and install the dependencies with `yarn install:ios`
```

**Review by TriplEight — DISMISSED** [review]

**Comment by Tbaut** [comment]
Not sure why this E2E test is hanging for hours and timing out... It failed in the last merge to master. I don't think it's related to this PR, but we should take a look.

**Pull request merged**